packages/commonwealth/server/controllers/server_communities_methods/create_community.ts

```diff
@@ -125,13 +122,7 @@ export async function __createCommunity(
   }
 
   if (community.icon_url) {
-    const iconFileSize = await getFileSizeBytes(community.icon_url);
-    if (iconFileSize === 0) {
-      throw new AppError(Errors.ImageDoesntExist);
-    }
-    if (iconFileSize / 1024 > MAX_IMAGE_SIZE_KB) {
-      throw new AppError(Errors.ImageTooLarge);
-    }
+    await checkUrlFileSize(community.icon_url, MAX_COMMUNITY_IMAGE_SIZE_BYTES);
```

Comment:
this can also be encapsulated -> MAX_COMMUNITY_IMAGE_SIZE_BYTES

Reply:
maybe use it as the default value of parameter maxsize in check....

Reply:
Encapsulated it. I think using it as a default is a bit overfitting since the function is meant to potentially be used in different contexts.